src/services/member/plugins/action/test/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTriggers, DiscriminatedItem } from '@graasp/sdk';
+
+import { AppDataSource } from '../../../../../plugins/datasource';
+import { Action } from '../../../../action/entities/action';
+import { saveActions } from '../../../../action/test/fixtures/actions';
+import { ItemTestUtils } from '../../../../item/test/fixtures/items';
+import { saveMember } from '../../../test/fixtures/members';
+
+const testUtils = new ItemTestUtils();
+const rawRepository = AppDataSource.getRepository(Action);
+
+export const getDateBeforeOrAfterNow = (dateDiff) => {
```

Comment:
How can this be before? You use a negative `dateDiff`? Not sure about this, I didn't see where you use it.

Reply:
basically I'm using it to specify start and end date for testing purposes, either the date or days difference can be a positive value to get days after or a negative value to get days before now.